Description: SCTP workaround for kfreebsd
 Freebsd has SCTP in the kernel but no sctp library is available,
 then iperf3 fails to compile because the lack of the library.
 .
 This patch moves the search for the library before kernel headers checks
Author: Roberto Lumbreras <rover@debian.org>
Forwarded: no
Index: iperf3-3.9/configure.ac
===================================================================
--- iperf3-3.9.orig/configure.ac	2020-10-04 23:11:23.362816143 +0200
+++ iperf3-3.9/configure.ac	2020-10-04 23:13:39.595787847 +0200
@@ -116,9 +116,10 @@
 # Check for SCTP support
 if $try_sctp; then
 AC_CHECK_HEADERS([sys/socket.h])
+AC_SEARCH_LIBS(sctp_bindx, [sctp])
+if test "x$ac_cv_search_sctp_bindx" != "xno"; then
 AC_CHECK_HEADERS([netinet/sctp.h],
 		 AC_DEFINE([HAVE_SCTP_H], [1], [Have SCTP support.])
-		 AC_SEARCH_LIBS(sctp_bindx, [sctp])
 		 AC_CHECK_TYPES([struct sctp_assoc_value], [], [],
 				[[#include <netinet/sctp.h>]]),
 		 [],
@@ -127,6 +128,7 @@
 #endif
 ])
 fi
+fi
 
 AC_CHECK_HEADER([endian.h],
 		AC_DEFINE([HAVE_ENDIAN_H], [1], [Define to 1 if you have the <endian.h> header file.]),
